pkgs/by-name/ia/ianny/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, stdenv
```

Comment:
Could you please clean unused arguments ?

Reply:
Why did you marked this as resolved?

Reply:
Sorry, it seems I forgot to push my changes.